charts/horizon/values.yaml

```diff
@@ -385,7 +385,7 @@ conf:
 
         CACHES = {
             'default': {
-                'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
+                'BACKEND': 'django.core.cache.backends.memcached.PyMemcacheCache',
```

Comment:
Consider long term maintaince, this change charts should go to openstack-helm like [1] 
And update [2]. Also If base version need to be bump, bump [3] before do [2].
After this, you should be able to run `hack/sync-charts.sh` and merge the change in after you clean up other parts.


Another way to do this without these change is to update `roles/horizon/vars/main.yml` and override the values.

[1] https://review.opendev.org/c/openstack/openstack-helm/+/899809
[2] https://github.com/vexxhost/atmosphere/blob/69ca2857cf5dea8863a91c87ba1fb37bdfa23e95/hack/sync-charts.sh#L79
[3] https://github.com/vexxhost/atmosphere/blob/69ca2857cf5dea8863a91c87ba1fb37bdfa23e95/vendir.yml#L146

Reply:
Thanks, Rico. It seems the initial request was to do the change in our own code but now I understand I should have submitted it upstream.